package.json

```diff
@@ -1,14 +1,22 @@
 {
+  "private": true,
   "name": "storybook",
   "version": "3.0.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/storybooks/storybook.git"
   },
+  "workspaces": [
+    "addons/*",
+    "app/*",
+    "lib/*",
+    "examples/cra-kitchen-sink",
+    "examples/vue-kitchen-sink"
+  ],
   "scripts": {
     "bootstrap": "./scripts/bootstrap.js",
-    "bootstrap:core": "lerna bootstrap --concurrency 1 --npm-client=\"yarn\" --hoist && node ./scripts/hoist-internals.js",
-    "bootstrap:docs": "cd docs && yarn install",
+    "bootstrap:core": "yarn install && lerna run prepublish && node ./scripts/hoist-internals.js",
```

Comment:
Unfortunately, at the moment yarn doesn't run lifecycle hooks when bootstrapping workspaces: https://github.com/yarnpkg/yarn/issues/3911
That's why we need `lerna run prepublish` here